src/features/organizations/components/ActivistPortlHeader/index.tsx

```diff
@@ -78,7 +83,15 @@ const ActivistPortalHeader: FC<Props> = ({
         >
           {topLeftComponent}
           {user && (
-            <>
+            <Box>
+              {isOfficial && (
+                <ZUIButton
+                  label="Organize"
+                  onClick={() => {
+                    router.push(`/organize`);
+                  }}
```

Comment:
I think we should use `href` for situations like this, because we want the button to behave like a link in the browser (e.g. open in new tab when ctrl+clicking).

Reply:
Fixed in 9a5a4faebea75f4442852538ab6b34aafcd0db83.